Source/JavaScriptCore/builtins/IteratorHelpers.js

```diff
@@ -54,10 +54,10 @@ function performIteration(iterable)
 }
 
 @linkTimeConstant
-function wrappedIterator(iterator)
+function wrapIterator(iterator)
```

Comment:
ditto (we can write following comment here?)

```
        // Since for-of loop once more looks up the @@iterator property of a given iterable,
        // it could be observable if the user defines a getter for @@iterator.
        // To avoid this situation, we define a wrapper object that @@iterator just returns a given iterator.
```